src/Package.appxmanifest

```diff
@@ -62,8 +69,11 @@
               <DevHomeProvider>
                 <Activation>
                   <CreateInstance ClassId="426A52D6-8007-4894-A946-CF80F39507F1" />
+                  <CreateInstance ClassId="F8B26528-976A-488C-9B40-7198FB425C9E" />
                 </Activation>
-                <SupportedInterfaces></SupportedInterfaces>
+                <SupportedInterfaces>
+                  <ComputeSystem />

```

Comment:
Why does Dev Home core need to register for an extension provider?

Reply:
If we don't then the Hyper-v extension won't actually show up. Dev Home has to be an extension for it to show up. Unless I'm missing something.